packages/cronie.rb

```diff
@@ -83,11 +91,11 @@ def self.postinstall
     puts 'sudo crontab -l'.lightblue
     puts
     puts 'To start the crond daemon at login, execute the following:'.lightblue
-    puts "echo '# start the crond daemon' >> ~/.bashrc".lightblue
-    puts "echo 'if [ -f #{CREW_PREFIX}/bin/startcrond ]; then' >> ~/.bashrc".lightblue
-    puts "echo '  #{CREW_PREFIX}/bin/startcrond' >> ~/.bashrc".lightblue
-    puts "echo 'fi' >> ~/.bashrc".lightblue
-    puts 'source ~/.bashrc'.lightblue
+    puts "echo '# start the crond daemon' >> #{CREW_PREFIX}/etc/env.d/profile".lightblue
+    puts "echo 'if [ -f #{CREW_PREFIX}/bin/startcrond ]; then' >> #{CREW_PREFIX}/etc/env.d/profile".lightblue
+    puts "echo '  #{CREW_PREFIX}/bin/startcrond' >> #{CREW_PREFIX}/etc/env.d/profile".lightblue
+    puts "echo 'fi' >> #{CREW_PREFIX}/etc/env.d/profile".lightblue
+    puts "source #{CREW_PREFIX}/etc/env.d/profile".lightblue
```

Comment:
This won't work and should be removed.  Prompt to `source ~/.bashrc` instead?

Reply:
Would it not make more sense to source `~/.profile`? I'm a little confused about how exactly the crew profile stuff works.